lib/plug/static.ex

```diff
@@ -178,7 +177,7 @@ defmodule Plug.Static do
     h in only or match?({0, _}, prefix != [] and :binary.match(h, prefix))
   end
 
-  defp serve_static({:ok, conn, file_info, path}, segments, gzip?, brotli?, qs_cache, et_cache, et_generation, headers) do
+  defp serve_static({:ok, conn, file_info, path}, segments, %{qs_cache: qs_cache, et_cache: et_cache, et_generation: et_generation, gzip?: gzip?, brotli?: brotli?, headers: headers}) do
```

Comment:
@josevalim I'm not sure about this long line, let me know if you want to split it.